README.rst

```diff
@@ -1,19 +1,25 @@
 .. image:: https://translate.fedoraproject.org/widgets/dnf/-/dnf-master/svg-badge.svg
     :alt: Translation status
     :target: https://translate.fedoraproject.org/engage/dnf/?utm_source=widget
-###############
- Dandified YUM
-###############
+
+==================================================================================================================================================================================================
+⚠️ **Notice:** This repository is for **DNF4** only. The current DNF version (DNF5), has its `own repo <https://github.com/rpm-software-management/dnf5>`_.
```

Comment:
I am also not an expert in English but I think the comma after `..version (DNF5), has..` should not be there -> `..version (DNF5) has..`.

Otherwise I think it looks good.

Reply:
You are right, sorry I missed that in my first review.

Reply:
done